x-pack/plugins/fleet/server/saved_objects/index.ts

```diff
@@ -393,13 +392,25 @@ const getSavedObjectTypes = (
       },
     },
   },
+  [MESSAGE_SIGNING_KEYS_SAVED_OBJECT_TYPE]: {
+    name: MESSAGE_SIGNING_KEYS_SAVED_OBJECT_TYPE,
+    hidden: true,
+    namespaceType: 'agnostic',
+    management: {
+      importableAndExportable: false,
+    },
+    mappings: {
+      properties: {
+        private_key: { type: 'binary' },
+        public_key: { type: 'binary' },
+        passphrase: { type: 'binary' },
+      },
+    },
```

Comment:
I assume you won't be searching on these binary fields, right?

In that case, let's not increase the field count unnecessarily, and just use:

```ts
    mappings: {
      dynamic: false,
    },
```

Reply:
good point, will update